runtime/README.md

```diff
@@ -1,11 +0,0 @@
-# Cadence Runtime
```

Comment:
What is the reason for removing this README?

Reply:
Well, isn't it out of date? `go run ./cmd` results in `go run: cannot run non-main package`, and you don't seem to use `antlr` for parser generation anymore.

Reply:
Yep, it's outdated an needs to be updated